packages/js/product-editor/src/blocks/description/edit.tsx

```diff
@@ -19,6 +19,16 @@ import { ModalEditor } from '../../components/modal-editor';
  * Internal dependencies
  */
 
+function clearDescription( blocks: BlockInstance[] ) {
```

Comment:
How about calling this `clearDescriptionIfEmpty`?

Reply:
Yep I was thinking about this or something similar but I could not find a good name for this. I also considered IfEmpty but since blocks list is not empty it was a little confusing for me. It should be something like remove the first block if its content is empty and it's just the only one block in the block list. Can you resume this in a couple of words?

Reply:
done -> [4373e02](https://github.com/woocommerce/woocommerce/pull/39229/commits/4373e021566b3a394ce8087309c133a41a4ef69d)